apis/userapi/userapi.yaml

```diff
@@ -1799,6 +1891,28 @@ definitions:
             type: string
             description: userId
 
+  MigrateUserReq:
+    description: request body for migrating user
+    type: object
+    properties:
+      request:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: userId of migrating user
+          action:
+            type: string
+            description: accept/reject only
+          feedId:
+            type: string
+            description: notificationId //optional
```

Comment:
Expand description `Pass the feed identifier by which the user is getting migrated. Empty feed identifiers are accepted, but this may result in same feed shown twice.`
Remove mention of optional.

Reply:
done